lib/active_fedora/indexing.rb

```diff
@@ -87,15 +87,17 @@ def index_config
         # @param [Boolean] softCommit - Do we perform a softCommit when we add the to_solr objects to SolrService. Default true.
         # @param [Boolean] progress_bar - If true output progress bar information. Default false.
         # @param [Boolean] final_commit - If true perform a hard commit to the Solr service at the completion of the batch of updates. Default false.
-        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false)
+        def reindex_everything(batch_size: 50, softCommit: true, progress_bar: false, final_commit: false) # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
Huh, I wonder why it worked then, as a "single line with an end-of-line comment."? But okay, I'll quick change it to the disable/enable pair you are asking for. And then I'm done with this for a while! If this isn't good enough (which might be reasonable), I'd ask someone with commit rights to do me a favor and revert #1219 which is buggy without this.